website/content/en/highlights/2025-08-12-0-49-0-upgrade-guide.md

```diff
@@ -46,8 +46,7 @@ CEF:1|Security|threatmanager|1.0|100|worm successfully stopped|10| dst=2.1.2.2 m
 
 #### Previous Behavior: Runtime Error
 
-If an input with space added to parse_cef was provided, it would result in a runtime error due to the inability to parse the line
-successfully.
+If an input for `parse_cef` includes spaces (` `), the line cannot be parsed correctly and results in a runtime error.
```

Comment:
```suggestion
If an input for `parse_cef` included spaces (` `), the line could be parsed correctly and resulted in a runtime error.
```

I know we already got feedback from docs on this but I feel like this should be in the past since it represents "Previous Behavior", just like in the previous block